### ISSUE TITLE
Exception when trying to download not-existing image

### ISSUE THREAD

**Issue by clemens-schulz** [jiaweihli/manga_downloader#64]
To reproduce run `./manga.py "Apocalypse no Toride"`, select "(4) MangaHere" and try to download the first chapter.
After a few seconds the download threat crashes:
```
Exception in thread Thread-3:1:   0% |                         | ETA:  --:--:--
Traceback (most recent call last):
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/Users/Clemens/Downloads/manga_downloader-master/src/parsers/base.py", line 375, in run
    raise FatalError("Thread crashed while downloading chapter: %s" % str(exception))
FatalError: Thread crashed while downloading chapter: local variable 'i' referenced before assignment
```

The problem seems to be in `parsers/base.py` on line 175/184. `i = urllib2.urlopen(request)` throws an exception and the exception handler tries to use the unassigned `i`. The requested url returns 404.

**Comment by clemens-schulz** [comment]
I've created a pull request with a fix.

**Comment by clemens-schulz** [comment]
Fixed with pull request #65
